checker/lib/README.developers

```diff
@@ -0,0 +1,11 @@
+Please read the README file in this directory first, as it is the file
+that is shipped in the distribution and is intended for users of the
+Checker Framework.
+
+When adding a .jar file to this directory, update the list in README and also
```

Comment:
The instructions tell me where to put a new jar file, but not how to build one. 
If I have my own library, how do I create a jar file for inclusion here? 
For what type systems are these libraries annotated? If I annotated BCEL with a different type system, would it also go here?